projects/observability/src/shared/graphql/request/builders/specification/entity/entity-specification-builder.ts

```diff
@@ -31,7 +38,12 @@ export class EntitySpecificationBuilder {
     return `entity_${idKey}_${nameKey}_${entityType === undefined ? 'unknownType' : entityType}`;
   }
 
-  private buildGraphQlSelections(idKey: string, nameKey: string, withEntityType: boolean): GraphQlSelection[] {
+  private buildGraphQlSelections(
+    idKey: string,
+    nameKey: string,
+    withEntityType: boolean,
+    additionalAttributes?: string[]
```

Comment:
nit, could avoid the `??` below:
```suggestion
    additionalAttributes: string[] = []
```